### PR TITLE
fix: cannot find remote branch

### PR DESCRIPTION
**Pull request by alichherawalla**
Hi, 

Thank you for this amazing library! In terms of getting a fingerprint of the project and its structure, there is nothing that I have come across that is easier to use. 

While running this action in one of my repos I faced the following issue which this PR resolves

```
/usr/bin/git rev-parse --verify <branch-name>
fatal: Needed a single revision
```

and there after pushing fails since it tries to push to a branch that already exists

```
  /usr/bin/git push --set-upstream origin <branch-name>
  To https://github.com/<org>/<repo-name>
   ! [rejected]        visualizer -> visualizer (non-fast-forward)
  error: failed to push some refs to 'https://github.com/<org>/<repo-name>'
  hint: Updates were rejected because the tip of your current branch is behind
  hint: its remote counterpart. Integrate the remote changes (e.g.
  hint: 'git pull ...') before pushing again.
  hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Changing to git switch with the --track flag solves the problem.

### CODE REVIEW

**Comment by alichherawalla** [comment]
This PR addresses and solves [this](https://github.com/githubocto/repo-visualizer/issues/42) issue

**Comment by alichherawalla** [comment]
@Wattenberger  If possible please take a look at this PR

**Comment by alichherawalla** [comment]
possible to please get a review here?

File: index.js [hunk 1]
```diff
@@ -26275,34 +26275,33 @@ var main = async () => {
   const maxDepth = core.getInput("max_depth") || 9;
   const customFileColors = JSON.parse(core.getInput("file_colors") || "{}");
   const colorEncoding = core.getInput("color_encoding") || "type";
-  const commitMessage = core.getInput("commit_message") || "Repo visualizer: updated diagram";
+  const commitMessage = core.getInput("commit_message") || "Repo visualizer: update diagram";
```
Comment (Wattenberger) [inline index.js:26278]: you'll want to make this change in src/index.jsx:30

**Review by Wattenberger — CHANGES_REQUESTED** [review]
this is great, thanks for the PR! sorry about the delay, we've been focusing on other projects at the moment. If you want to move the commit message name change to the uncompiled file, I can merge after!

**Review by Wattenberger — APPROVED** [review]
ah, I see! that's from a previous change - we're all good!

**Pull request merged**